### PR TITLE
Release 9.5.0

### PR DESCRIPTION
**Pull request by hy-ca**

### CODE REVIEW

File: index-fr.md [hunk 1]
```diff
@@ -12,8 +12,8 @@ lang: fr
 		<p>Les gabarits et les conceptions communes si dessous sont une référence d'implémentation du <a href="https://conception.canada.ca">Système de conception de Canada.ca</a>, incluant les exigences obligatoire de la spécifications du contenu et de l’architecture de l'information (C&amp;AI) pour Canada.ca. Les ministères et organisme du gouvernement du Canada peuvent y contribuer en publiant leur modèle et leur conception commune via le <a href="https://github.com/wet-boew/GCWeb">dépôt github de GCWeb</a>.</p>
 	</div>
 	<div class="col-xs-12 col-md-auto pull-right">
-		<p><a href="https://github.com/wet-boew/GCWeb/archive/v9.4.1.zip" class="btn btn-primary">Télécharger le thème <strong>GCWeb v9.4.1</strong></a><br />
-			<small>(<time>{{ page.dateModified | %F }}</time> - <a href="https://github.com/wet-boew/gcweb/releases/tag/v9.4.1">Note de version</a>)</small></p>
+		<p><a href="https://github.com/wet-boew/GCWeb/archive/v9.5.0.zip" class="btn btn-primary">Télécharger le thème <strong>GCWeb v9.4.1</strong></a><br />
```
Comment (GormFrank) [inline index-fr.md:15]:
```suggestion
		<p><a href="https://github.com/wet-boew/GCWeb/archive/v9.5.0.zip" class="btn btn-primary">Télécharger le thème <strong>GCWeb v9.5.0</strong></a><br />
```

File: index-en.md [hunk 2]
```diff
@@ -12,8 +12,8 @@ lang: en
 		<p>The page templates and design patterns below comprise a reference implementation of the <a href="https://design.canada.ca">Canada.ca design system</a>, including the mandatory requirement of the Content and Information Architecture (C&amp;IA) Specification. Government of Canada departments and agencies can contribute additional patterns and templates via <a href="https://github.com/wet-boew/GCWeb">GCWeb github repository</a>.</p>
 	</div>
 	<div class="col-xs-12 col-md-auto pull-right">
-		<p><a href="https://github.com/wet-boew/GCWeb/archive/v9.4.1.zip" class="btn btn-primary">Download GCWeb theme <strong>v9.4.1</strong></a><br />
-			<small>(<time>{{ page.dateModified | %F }}</time> - <a href="https://github.com/wet-boew/gcweb/releases/tag/v9.4.1">Release notes</a>)</small></p>
+		<p><a href="https://github.com/wet-boew/GCWeb/archive/v9.5.0.zip" class="btn btn-primary">Download GCWeb theme <strong>v9.4.1</strong></a><br />
```
Comment (GormFrank) [inline index-en.md:15]:
```suggestion
		<p><a href="https://github.com/wet-boew/GCWeb/archive/v9.5.0.zip" class="btn btn-primary">Download GCWeb theme <strong>v9.5.0</strong></a><br />
```

**Review by GormFrank — CHANGES_REQUESTED** [review]
Also please bump the version of WET-BOEW as soon as WET-BOEW is released. Thanks!

File: index-fr.md (hunk 1 above)
Comment (hy-ca) [reply index-fr.md:15]: As discussed, required change has been completed. The commit and PR will be updated/amended once WET-BOEW is released (wet-boew commit sha in package-lock.json can be updated by then).

File: index-en.md (hunk 2 above)
Comment (hy-ca) [reply index-en.md:15]: As discussed, required change has been completed. The commit and PR will be updated/amended once WET-BOEW is released (wet-boew commit sha in package-lock.json can be updated by then).

File: index-fr.md (hunk 1 above)
Comment (hy-ca) [reply index-fr.md]: WET-BOEW was released. WET-BOEW version info and commit sha were updated in package.json and package-lock.json files.

File: index-en.md (hunk 2 above)
Comment (hy-ca) [reply index-en.md]: WET-BOEW was released. WET-BOEW version info and commit sha were updated in package.json and package-lock.json files.

**Review by GormFrank — APPROVED** [review]

**Pull request merged**